### PR TITLE
[JDK17&21 on Release Test] LPD-29624 Use jdk21_zulu for release tester test batches

### PR DESCRIPTION
**Pull request by julschong**
@vicnate5 @tinatian 
I changed all portal-release test batches to use jdk21_zulu **except**:
- batches using linux-images like amazonlinux2
- `patching-tool-mysql57-jdk8` - I don't think this starts server so I didn't change it
- `playwright-js-bundle-tomcat90-mysql57-jdk8` - not sure if I should change this too

2 types of changes in this PR:
- `Add functional...` commits: This means that the jdk8 test batch is used somewhere else other than portal-release, so I cannot just replace it with jdk21_zulu. Have to keep jdk8 and add new jdk21_zulu test batch.
- `Convert functional...` commits: This means that the jdk8 test batch is only used in portal-release, so I can directly replace it.

I asked ci team to help run job summary again to make sure

### CODE REVIEW

**Comment by vicnate5** [comment]
@julschong Can you try out the playwright regular batch (not the `-bundle-` one) and see what happens on CI?

We should change that one if possible and if it actually takes effect, that batch is all the new functional tests from all teams.

File: test.properties [hunk 1]
```diff
@@ -7669,22 +7686,20 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
...
-        functional-bundle-tomcat-postgresql121-jdk8,\
+        functional-bundle-tomcat-mysql80-jdk21_zulu,\
+        functional-bundle-tomcat-postgresql121-jdk21_zulu,\
         functional-bundle-ubuntu18-tomcat90-mysql57-jdk8,\
-        functional-smoke-bundle-wildfly180-mysql57-jdk8,\
-        functional-smoke-bundle-wildfly230-mysql57-jdk8,\
-        functional-smoke-bundle-wildfly261-mysql57-jdk8,\
+        functional-smoke-bundle-wildfly180-mysql57-jdk21_zulu,\
+        functional-smoke-bundle-wildfly230-mysql57-jdk21_zulu,\
+        functional-smoke-bundle-wildfly261-mysql57-jdk21_zulu,\
```
Comment (julschong) [inline test.properties:7702]: these didn't show in job summary

File: test.properties [hunk 2]
```diff
@@ -7669,22 +7686,20 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
...
         patching-tool-mysql57-jdk8,\
         playwright-js-bundle-tomcat90-mysql57-jdk8
```
Comment (julschong) [inline test.properties:7704]: @vicnate5 do you mean to change this to `playwright-js-tomcat90-mysql57-jdk21_zulu` /.

**Comment by julschong** [comment]
job summary result: https://test-5-1.liferay.com/userContent/jobs/test-portal-release/builds/889/job-summary/index.html

the wildfly smoke test did not show, need to find out why

File: test.properties (hunk 1 above)
Comment (julschong) [reply test.properties:7702]: @vicnate5 in [old release test job summary](https://ffa45b50ebdf44555fdc85cdffd74f207ce435b75a272acb63d3f66-apidata.googleusercontent.com/download/storage/v1/b/testray-results/o/2024-07%2Ftest-1-1%2Ftest-portal-release%2F4450%2Fjob-summary%2Findex.html.gz?jk=AYBlUPBdAE_OhsnalfL_cFbh7EilBTE5f_rbYtPBJV0Qnz7idYXuEo8qcG66gJeXJFm8RMzcCOrBXCXz_73fBcCZuTDKN70Svcpc_pt2JTO1lwqLD2yEgvaSYQ_JZ4KBAh390kya7lcGo_PENW9ySwM2cLOz7GXQZF1kQu0FR3PdrYld4OXDKPqrId_jqsMqAb5QagZALUSaQ2dAGr2T71BUKqydU0fxegPGHVGCV-Lnzb8ZUpOaEuRhHPgIlbYn2-cFWqNUcLTZoIf7f2hLq5O50w4Yf-r9Xf3_OTjrIUsYV93G-lwJYLlLeV4UzIXi-Tn9W96hNOyztuhbiPSeEqzQQrorAEPJFVpY1ReF4Ic4qXTUBIrQQUDQ8zktgyHHhEmp5uKpleDza-wypXJ0JFKHKcoZcbrR7b4XcudqFSCv9yHt_eWLMUccHrvCnE_9aI8E78_t8Tr7qSTBzNhm5b2mjmkO-fmr1s90I2Z5PhZhaNfwwa-DbolINL8ZyiQSk-6RqcpHo1sTapkv_TUsa9Kb9ZfRrtzc4SOVSNlvhSKbvP7V4iqZqsjbxSgnyO5K1Tv8QrcezKiJcoZk2zDWgxKdbaQlBIBlghaFod0fOlfr6uEaIrHg80qOXTp1TiyHJ5KRNfvKoDrhdcA-DgYnkgjYa0rjcPiJ21-QyCuxhIjXikEjj1cpBBIMQE0BKdGChuVryZOTihJ3FZMQ_l32nNvMVwtS_fwN2hJySSzVKsFUD_mxeUv_vdCC28MeFzMTTpbG6jsa1Bf69fXGtyIU8IHflQ5zkJP-_jnvU6vES6DiQjaxwXUSikCz0DiuqOChqCcM2gkVU1FrKsAL-ncQr1k8_3HPSmq-2hwHq6utUxhGSh46x90S4U94tmwDcdH8QhsbUYnxBeq-tzsdRhqUOjo-LwCcUvppv_6zERYWhvG9_uEXAw5NoyW6z2Kkzl1sy62TGanBQDgMFd2itOX0Vmyl_jFEVML4qOQXLW1krrGg1NIc9EY25ZSSGLmJDRBRx7-Vh77Nxubui2trS83cbJPCzLC_6I6FUuYQPHp7OTPHeP0pho9o94wAek4iYxtNxPZ8Jc6yKkdEd81NFb1qfx1HV4_TfM3ixMEbcatrxn8dMckPcSgEru9j-Nc4Iq6iz5g7PIxFJu2AcLoEndbb5yruxyRHLLnmdV-q4tzleEnW_mPkSXUF2tizJsHVS00-D2zoq_FCRBh_JScL6KWQldnj7E42rWPz5nsN7pvkKvRL3V2bRw&isca=1)
I don't see these smoke wildfly test batches run either. Maybe no test match the query criteria?

File: test.properties (hunk 2 above)
Comment (vicnate5) [reply test.properties:7704]: @julschong Yes

File: test.properties (hunk 1 above)
Comment (vicnate5) [reply test.properties:7702]: Gotta check the query.

We also should remove wildfly 18 and 23, but that should be a new ticket at this point.

**Comment by julschong** [comment]
added playwright commit: [e9511bf](https://github.com/liferay-core-infra/liferay-portal/pull/7316/commits/e9511bfe38823b5fc6ae166e66e1a85b56b99a6b)

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
@vicnate5 besides job-summary making sure that I didnt make a typo somewhere, is there any other way to check that the test batches are now actually running successfully using jdk21_zulu?

File: test.properties (hunk 1 above)
Comment (vicnate5) [reply test.properties:7702]: @julschong this can be covered by https://liferay.atlassian.net/browse/LPD-31337

**Comment by vicnate5** [comment]
@julschong The only way to confirm it is working that I know if is to trigger the release tester (but it is heavy). You could ask CI team how they would prefer to handle this.

**Comment by julschong** [comment]
@vicnate5 I have a plan:
Since this PR has more batch changes, I'd like release team to not include this PR's changes in their test yet. It would be good to see if the changes Ive made on other pulls so far cause any issue on their end.
I'm assuming this week they will run the release tester to analyze result since the last time was on 7/25.
CI team said we can trigger a full test on friday with this PR change since release test is heavy.

The only thing I want to merge in right now is this commit, I can send separately: https://github.com/liferay-core-infra/liferay-portal/pull/7316/commits/c1852bcf79c6db31b0e80d9ad1acc0deed829869

**Comment by julschong** [comment]
@vicnate5 close this PR for now. will send a new one for https://github.com/liferay-core-infra/liferay-portal/commit/c1852bcf79c6db31b0e80d9ad1acc0deed829869

**Comment by julschong** [comment]
see https://github.com/liferay-core-infra/liferay-portal/pull/7318

**Pull request closed without merging**